<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateClothoidParallel</class>
 <widget class="QDialog" name="CreateClothoidParallel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create-Parallel-Clothoids</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="lineEdit_distance">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Carriageway width =</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Step-distance of original roadpoints =</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="lineEdit_width">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="pushButtonGenerate">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This function draws two parallel lines around your selected clothoid.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Select clothoid by using the alignment-window on the right side.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QRadioButton" name="radioButton_sameA">
     <property name="text">
      <string>Same A (nearly exactly, with advantage of used clothoids) </string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QRadioButton" name="radioButton_exactly">
     <property name="text">
      <string>Exactly parallel roadpoints (please insert step-distance)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
